src/System.Data.SQLite/NativeMethods.cs

```diff
@@ -127,6 +127,56 @@ internal static class NativeMethods
 		[DllImport(c_dllName, CallingConvention = c_callingConvention)]
 		public static extern int sqlite3_trace_v2(SqliteDatabaseHandle db, SQLiteTraceEvents eventsMask, SQLiteTraceV2Callback callback, IntPtr userData);
 
+#if NET5_0
+		static NativeMethods()
+		{
+			var assembly = typeof(NativeMethods).Assembly;
+			var configFile = System.IO.Path.GetFileName(assembly.Location) + ".config";
+			if (System.IO.File.Exists(configFile))
+			{
+				try
+				{
+					using var stream = new System.IO.FileStream(configFile, System.IO.FileMode.Open);
+					using var reader = System.Xml.XmlReader.Create(stream);
+					var dllMapConfig = System.Xml.Linq.XDocument.Load(reader);
+
+					
+					foreach (var element in dllMapConfig.Root.Elements())
+					{
+						if (element.Name != "dllmap")
+							continue;
+
+						if (element.Attribute("dll").Value == c_dllName)
+						{
+							s_mappedDllName = element.Attribute("target").Value;
+							break;
+						}
+					}
+
+					if (!string.IsNullOrEmpty(s_mappedDllName))
+						NativeLibrary.SetDllImportResolver(typeof(NativeMethods).Assembly, DllImportResolver);
+				}
+				catch (Exception)
+				{
+				}
```

Comment:
It's really only required b/c we add `cerod` support to our version of the assembly. On macOS this library works fine and I didn't notice the issue on Proclaim. I'm happy to let it throw here, but it felt strange to do since the `.dll.config` isn't shipped with this NuGet package.

Reply:
Since we explicitly check if the file exists, it would presumably only throw if the config file wasn't valid XML, which seems reasonable to me.

Reply:
Fair enough; removed.